simplifyDriver.cc

```diff
@@ -82,859 +87,232 @@ using namespace std;
 //! If you wish not to see the output, change the value: verbose = false;
 bool verbose=true;
 
+// Examples
+void CSR_Gauss_Seidel_Example();
+void CSR_ILU_Example();
 
+// Utility function
+bool printRelation(string msg, Relation *rel);
 void EXPECT_EQ(string a, string b);
+void EXPECT_EQ(Relation *a, Relation *b);
+int str2int(string str);
 
-
-int main(int argc, char **argv)
+// Reads information from a JSON file (inputFile), and applies
+// the simplfication algorithm to the sets found in the file. 
+void simplifiy(string inputFile)
 {
-    // (1)
```

Comment:
Where did all of this go?  Into another test case?  We definitely don't want to throw old test cases away even if we have automated them better.